AsyncStreamReaderLoggingExample.py

```diff
@@ -0,0 +1,84 @@
+# Tomer Heber's coding challenge 20 Oct 2021.
+
+import asyncio
+import argparse
+import http.client
+import json
+import logging
+import time
+
+from abc import ABC, abstractmethod
+from datetime import datetime
+
```

Comment:
our default format is 

```
if __name__ == '__main__': 
    args  = parse_args() 
    main( args.username, args.password, args.site) 
```

Please make that small rebuild /remodel of how this is launched/run.

Reply:
refactored as requested.